packages/brokeneck-react/src/components/Provider.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { screen, render } from '@testing-library/react'
+
+import RootContext from '../components/RootContext'
+
+import Provider from './Provider'
+
+describe('Provider', () => {
+  it('should render Provider component with a known provider', () => {
+    render(
+      <RootContext.Provider value={{ provider: { name: 'cognito' } }}>
+        <Provider />
+      </RootContext.Provider>
+    )
+
+    expect(screen.queryByTestId('provider-icon')).toBeInTheDocument()
+    expect(screen.queryByText('cognito')).toBeInTheDocument()
+  })
+
+  it('should render Provider component with an unknown provider', () => {
+    render(
+      <RootContext.Provider value={{ provider: { name: 'unknown' } }}>
+        <Provider />
+      </RootContext.Provider>
+    )
+
+    expect(screen.queryByTestId('provider-icon')).toBeNull()
```

Comment:
for more consistency

```suggestion
    expect(screen.queryByTestId('provider-icon')).not.toBeInTheDocument()
```

Reply:
Oh, that was a leftover from testing with getByTestId, which didn't seem to like this way of asserting the absence of the element. Updating it now.